gradle/libs.versions.toml

```diff
@@ -0,0 +1,23 @@
+[versions]
+gradle = "8.0.0"
+kotlin = "1.9.21"
+dokka = "1.9.10"
+ktlint = "11.3.1"
+detekt = "1.22.0"
```

Comment:
Can we just use detekt-formatting instead of ktlint like we do on the new project template?